README.md

```diff
@@ -79,7 +87,7 @@ import UAParser from 'ua-parser-js';
 
 ### Injection
 
-By default, this addon will generate an initializer in `app/initializers/user-agent.js` that injects the `userAgent` service app-wide. If the `userAgent` property conflicts with other addons or you wish to use manual injection (`Ember.service.inject`) you can override this file.
+Prior to `0.11.0`, this addon generated an initializer in `app/initializers/user-agent.js` that injected the `userAgent` service across all controllers, components and routes. This does not happen in `>=0.11.0`.
```

Comment:
```suggestion
Prior to `0.11.0`, this addon generated an initializer in `app/initializers/user-agent.js` that injected the `userAgent` service across all controllers, components and routes. This does not happen in `>=0.11.0`.

You can restore this behavior by manually performing these implicit injections (see [#42](https://github.com/willviles/ember-useragent/pull/42)), however this is highly discouraged, as this feature is deprecated by the upcoming Ember `v4.0`. If you were relying on these implicit injections, you should instead refactor your code to explicitly inject the `userAgent` service.
```

Reply:
You could include something like the following in the PR body:

Ember `v4.0` removes support for implicit inejctions, as perf https://github.com/emberjs/ember.js/pull/19680. If you were relying on this feature in your code, you should migrate your code base and explicitly inject the `userAgent` service at the call sites.

If you absolutely must temporarily restore the old behavior, you can include this intializer as `app/intializers/user-agent.js`:

https://github.com/willviles/ember-useragent/blob/32039b648e3e0eb18f99b5db74d9ce9dcfd4abfa/addon/initializers/user-agent.js#L7-L14

Please note, that you will still have to migrate your app to use explicit injections, in order to be able to upgrade to Ember `v4.0`, so this should only be used as a last-resort temporary band-aid.